### PR TITLE
Release v1.2.3

### PR DESCRIPTION
**Pull request by jathanism**
Request to merge `develop` into `main` for release v1.2.3.

### CODE REVIEW

**Review by glennmatthews — APPROVED** [review]
Don't forget to `poetry version 1.2.3`! :grin:

File: nautobot/docs/release-notes/version-1.2.md [hunk 1]
```diff
@@ -165,6 +165,16 @@ Just as with the UI, the `slug` can still always be explicitly set if desired.
 
 - [#650](https://github.com/nautobot/nautobot/issues/650) - Job Results UI now render job log messages immediately
 
+### Security
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:168]: To be consistent with our other release-notes, `Security` should come after `Fixed`.

Comment (bryanculver) [reply nautobot/docs/release-notes/version-1.2.md:168]: @jathanism If I'm reading this correctly it's adding the Security notes into the 1.2.2 section and not the 1.2.3 section?

Comment (glennmatthews) [reply nautobot/docs/release-notes/version-1.2.md:168]: Good catch!

Comment (jathanism) [reply nautobot/docs/release-notes/version-1.2.md:168]: Uh yeah I put it in the wrong section entirely. Whoops.

**Review by bryanculver — APPROVED** [review]
LGTM

:shipit:

**Pull request merged**